docs/source/conf.py

```diff
@@ -15,22 +15,51 @@
 import os
 import sys
 
+from astropy.io.misc import yaml
+
+import simtools.version
+
 sys.path.insert(0, os.path.abspath("../../simtools"))
 sys.path.insert(0, os.path.abspath("../../applications"))
 sys.path.insert(0, os.path.abspath("../.."))
 
 
+def get_authors_from_citation_file(file_name):
+    """
+    Read list of authors from CITATION.cff file
+
+    """
+    try:
+        with open("../../CITATION.cff") as file:
+            citation = yaml.load(file)
+    except FileNotFoundError:
+        raise
+
+    author = ""
+    try:
+        for person in citation["authors"]:
+            author = author + person["given-names"] + " " + person["family-names"]
+            author += " (" + person["affiliation"] + "), "
+    except KeyError:
+        pass
+    return author[:-1]
+
+
 # -- Project information -----------------------------------------------------
 
 project = "gammasim-tools"
 copyright = "2022, gammasim-tools developers"
-author = "Raul R Prado and Orel Gueta and Gernot Maier and Victor Barbosa Martins"
+author = get_authors_from_citation_file("../CITATION.cff")
+rst_epilog = """
+.. |author| replace:: {author}
```

Comment:
I have never seen this way of replacing. Is this python or some weird RST way of replacing?

Reply:
This is weird sphinx stuff: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-rst_epilog